pkg/arvo/tests/sys/vane/ames.hoon

```diff
@@ -447,6 +485,92 @@
     !>  [~[/g/talk] %give %done `error]
     !>  (snag 1 `(list move:ames)`moves5)
 ::
+++  test-fine-request
+  ^-  tang
+  =/  want=path  /c/z/1/kids/sys
+  =^  moves1  nec  (call nec ~[/g/talk] %keen ~bud want)
+  =/  req=hoot:ames
+    %+  snag  0
+    %+  murn  ;;((list move:ames) moves1)
+    |=  =move:ames
+    ^-  (unit hoot:ames)
+    ?.  ?=(%give -.card.move)    ~
+    ?.  ?=(%send -.p.card.move)  ~
+    `;;(@uxhoot blob.p.card.move)
+  =/  =packet:ames  (decode-packet:ames `@ux`req)
+  ?<  sam.packet
+  ?>  req.packet
+  =/  twit
+   (decode-request:ames `@ux`content.packet)
+  ~&  twit
+  (expect-eq !>(1) !>(1))
+::
+++  test-fine-hunk
+  ^-  tang
+  %-  zing
+  %+  turn  (gulf 1 10)
+  |=  siz=@
+  =/  want=path  /~bud/0/1/c/z/(scot %ud siz)/kids/sys
+  ::
+  =/  =beam  [[~bud %$ da+now:bud] (welp /fine/hunk/1/16.384 want)]
+  =/  [=mark =vase]  (need (need (scry bud %x beam)))
+  =+  !<(song=(list @uxmeow) vase)
+  %+  expect-eq
+    !>(siz)
+    !>((lent song))
+::
+++  test-fine-response
+  ^-  tang
+  ::%-  zing
+  ::%+  turn  (gulf 1 50)
+  ::|=  siz=@
+  ::=/  want=path  /~bud/0/1/c/z/(scot %ud siz)/kids/sys
+  =/  want=path  /~bud/0/1/c/x/1/kids/app/dojo/hoon
+  =/  dit  (jam %hoon dojo)
+  =/  exp  (cat 9 (fil 3 64 0xff) dit)
+  =/  siz=@ud  (met 13 exp)
+  ^-  tang
+  ::
+  =/  =beam  [[~bud %$ da+now:bud] (welp /fine/hunk/1/16.384 want)]
+  =/  [=mark =vase]  (need (need (scry bud %x beam)))
+  =+  !<(song=(list @uxmeow) vase)
+  =/   paz=(list have:ames)
+    %+  spun  song
+    |=  [blob=@ux num=_1]
+    ^-  [have:ames _num]
+    :_  +(num)
+    =/  =meow:ames  (decode-response-packet:ames blob)
```

Comment:
These ames tests fail to build, with a stack trace pointing at this line.

Reply:
Fixed the tests.  Hopefully the CI will work now.